common/web/lm-worker/src/main/models/dummy-model.ts

```diff
@@ -48,6 +66,22 @@ export default class DummyModel implements LexicalModel {
     if (options.punctuation) {
       this.punctuation = options.punctuation;
     }
+
+    if (options.toKey) {
+      this.toKey = options.toKey;
+    }
+
+    if (options.wordbreaker) {
+      this.wordbreaker = options.wordbreaker;
+    }
+
+    if (options.applyCasing) {
+      this.applyCasing = options.applyCasing;
+    }
+
+    if (options.languageUsesCasing) {
+      this.languageUsesCasing = options.languageUsesCasing;
+    }
```

Comment:
Any reason why we can't just shorten this?

```suggestion
    this.toKey = options.toKey;
    this.wordbreaker = options.wordbreaker;
    this.applyCasing = options.applyCasing;
    this.languageUsesCasing = options.languageUsesCasing;
```

The only reason I can think of for not doing this -- are we enumerating over properties in such a way that this would fail? If so, that seems like a second problem waiting!

If you are concerned about nullish tests, then you could:

```suggestion
    this.toKey = options.toKey ?? undefined;
    this.wordbreaker = options.wordbreaker ?? undefined;
    this.applyCasing = options.applyCasing ?? undefined;
    this.languageUsesCasing = options.languageUsesCasing ?? undefined;
```